uniffi_macros/src/util.rs

```diff
@@ -212,6 +212,26 @@ mod kw {
 
 #[derive(Default)]
 pub(crate) struct CommonAttr {
+    /// Specifies the `UniFfiTag` used when implementing `FfiConverter`
+    ///   - When things are defined in the proc-macros, this is `None` which means create a blanket
```

Comment:
Not sure what terminology we are using for this in general, but I'd suggest s/in the/with/ ?